resources/views/includes/status.blade.php

```diff
@@ -118,6 +118,10 @@ class="progress-bar progress-time"
     </div>
     <div class="card-footer text-muted interaction">
         <span class="float-end like-text">
+            <i class="fas
+{{["fa-globe-americas", "fa-lock-open", "fa-user-friends", "fa-lock"][$status->visibility]}} visibility-icon text-small"
```

Comment:
Omg. xD But.. okay.. 

![photo](https://media.giphy.com/media/sR2YaENch4sog/giphy.gif)

Reply:
I mean... better than 12 lines of if-statements! 😂